pxtarget.json

```diff
@@ -95,6 +96,90 @@
         "breakBlock": true,
         "continueBlock": true
     },
+    "compile": {
+        "isNative": true,
+        "useUF2": true,
+        "webUSB": true,
+        "hasHex": true,
+        "saveAsPNG": true,
+        "deployDrives": ".*",
+        "deployFileMarker": "INFO_UF2.TXT",
+        "driveName": "DRIVE",
+        "openocdScript": "source [find interface/cmsis-dap.cfg]; set CHIPNAME at91samd21g18; source [find target/at91samdXX.cfg]",
+        "flashChecksumAddr": 8372,
+        "flashCodeAlign": 256,
+        "upgrades": [
+            {
+                "type": "package",
+                "map": {
+                    "robotis-cm": "robotis-cm-300"
+                }
+            }
+        ],
+        "patches": {
+            "0.0.0 - 0.6.8": [
+                {
+                    "type": "package",
+                    "map": {
+                        "arduino-mkr": "arduino-mkr1000"
+                    }
+                }
+            ]
+        }
+    },
+    "compileService": {
+        "buildEngine": "codal",
+        "codalTarget": {
+            "name": "missing",
+            "url": "https://github.com/microsoft/pxt-does-not-exist",
+            "branch": "v0",
+            "type": "git"
+        },
+        "codalBinary": "CIRCUIT_PLAYGROUND",
+        "yottaConfig": {
+            "pxt": {
+                "board": {
+                    "id": "BOARD_ID_CPLAY"
+                }
+            }
+        },
+        "githubCorePackage": "lancaster-university/codal",
+        "gittag": "v0.6.3",
+        "serviceId": "codal2cp",
+        "dockerImage": "pext/yotta:latest"
+    },
+    "variants": {
+        "cm300": {
+            "serial": {
+                "useHF2": true
+            },
+            "compile": {
+                "openocdScript": "source [find interface/cmsis-dap.cfg]; source [find target/nrf52.cfg]",
+                "flashChecksumAddr": 0,
+                "useUF2": true,
+                "webUSB": false,
+                "uf2Family": "0xada52840"                
+            },
+            "compileService": {
+                "codalTarget": {
+                    "name": "codal-cm300",
+                    "url": "https://github.com/ROBOTIS-GIT/codal-cm300.git",
+                    "branch": "develop",
+                    "type": "git"
+                },
+                "codalBinary": "CM300",
```

Comment:
are you sure this is the file that codal produces?

Reply:
Sure.

In fact, it has been used by converting CM300.hex to .uf2 in CODAL.
The configuration is also shown below. ([Link](https://github.com/ROBOTIS-GIT/codal-cm300/blob/573c3dae2c363e98bddd13070d981b7c539a5f39/target.json#L2))
```json
    "device":"CM300",
    "processor":"NRF52840",
```

Does this cause an error?

Reply:
Probably - try the original value.

Reply:
Done on 6a37860.